frontend/src/components/ModalForSearch.js

```diff
@@ -0,0 +1,19 @@
+import { Modal, Button } from 'react-bootstrap';
+
+function ModalForSearch(props) {
+  return (
+    <>
+      <Modal {...props}>
+        <Modal.Header closeButton>
+          <Modal.Title>Konyhatündér üzenet</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>Nincs ilyen hozzávalót tartalmazó recept</Modal.Body>
+        <Modal.Footer>
+          <Button onClick={props.onHide}>Close</Button>
```

Comment:
Ez is lehetne magyar, ha már a felület többi része az.

Reply:
javítva